src/index.ts

```diff
@@ -82,3 +85,28 @@ export const parseKind = (kind: string, { rootSeparator, groupSeparator }: Separ
     groups,
   };
 };
+
+/**
+ * csf story definition
+ */
+export interface CSFStory {
+  story?: {
+    /**
+     * story name if differnet from the export name
+     */
+    name?: string;
```

Comment:
Can we make this work the other way around? You always supply a human readable name and we create an valid export from the name? The actual export name is mostly noise.

Reply:
You have to provide an export name no matter what right? Do you mean the `storyId`?

Another approach would be to make the list of stories an array that you export (which would solve a separate ordering issue that's come-up elsewhere) but that's something for consideration in CSFv2 I guess ;)